wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -0,0 +1,73 @@
+# Livestreaming osu!
+
+This guide will show you how to livestream osu! on Twitch using [OBS Studio](https://obsproject.com/ "Open Broadcaster Software"). Doing so is a relatively simple, although it might require a reliable computer able to both run and livestream osu!.
+
+## Twitch
+
+### Creating an account
+
+Before you setup the streaming app, you will first need to create an account on Twitch. Proceed to [Twitch's website](https://twitch.tv) and click the `Sign Up` button in the top right - you will be prompted with a registration form. Fill in the fields and click `Sign Up` again.
+
+![](img/twitch-registration.png "Registrating on Twitch")
+
+Now you will be asked to enter a verification code that was sent on your email address. After that, your account will be accessible at `twitch.tv/<username>` and ready to go!
```

Comment:
I don't think this guide needs a walk through to sign up for a Twitch account. It can probably just be:

```md
If you have not already, sign up for a [Twitch account](https://www.twitch.tv/signup "Twitch").
```

If you do go with this, this section would then be short, so maybe just merge it in with the other section or get rid of both subheadings.